registry/handlers/images.go

```diff
@@ -415,14 +416,39 @@ func (imh *imageManifestHandler) applyResourcePolicy(manifest distribution.Manif
 
 // DeleteImageManifest removes the manifest with the given digest from the registry.
 func (imh *imageManifestHandler) DeleteImageManifest(w http.ResponseWriter, r *http.Request) {
-	ctxu.GetLogger(imh).Debug("DeleteImageManifest")
+	ctxu.GetLogger(imh).Debugf("DeleteImageManifest. Tag: %s Digest: %s", imh.Tag, imh.Digest)
```

Comment:
Use `WithFields` here to add the digest and tag, rather than formatting them into the error message. They might actually already be in the context.